cmd/set.go

```diff
@@ -49,6 +49,7 @@ func cmdSetRegistryCreds(clientset *client.ConfigSet) *cobra.Command {
 	}
 
 	setRegistryCredsCmd.Flags().StringVar(&rc.Host, "registry", "", "Registry host address")
+	setRegistryCredsCmd.Flags().StringVar(&rc.ProjectID, "project", "", "If set, use this value instead of the username in image URL: <host>/<project>/<image>")
```

Comment:
1. (nit) Image names are not "URLs" although they may look like one.
1. Maybe use this more concrete example?

```suggestion
	setRegistryCredsCmd.Flags().StringVar(&rc.ProjectID, "project", "", "If set, use this value instead of the username in image names. Example: gcr.io/<project>")
```